### PR TITLE
Added some services.

### PR DESCRIPTION
**Pull request by thispsj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -770,6 +768,15 @@ Table of Contents
...
   * [darklang.com](https://darklang.com/) - Hosted language combined with editor and infrastructure. Free during the beta, generous free tier planned after beta.
+  * [Firebase](https://firebase.com) — Firebase helps you build and run successful apps. Offers a variety of services in the Free Spark Plan listed here :
+    - Authentication - 10K Phone Auths/month (seperate 10K limit for US,Canada & India) & unlimited auths for other methods.
+    - Cloud Firestore,Cloud Functions & BigQuery - same as offered in GCP Always Free Tier.
+    - Hosting - 10GB storage with 360 MB data transfer per day with custom domain & SSL.
+    - Firebase ML - Custom Model Deployment.
+    - Realtime Database - 100 simultaneous connections with 1GB storage & 10GB downloads per month.
+    - Cloud Storage - 5GB storage with 1GB/day upto 50k/day downloads and 20k/day upload operations.
+    - Testlab - 10 Virtual Device & 5 Physical Device tests per day.
+    - Always Free :A/B Testing, Analytics, App Distribution, App Indexing,Cloud Messaging (FCM), Crashlytics, Dynamic Links, In-App Messaging, Performance Monitoring, Predictions, and Remote Config.
```
Comment (ripienaar) [inline README.md:779]: isnt this technically part of google cloud? I think it should be a condense one line by the existing big google cloud entry.

File: README.md [hunk 2]
```diff
@@ -789,7 +796,13 @@ Table of Contents
   * [ServiceNow Developer Program](https://developer.servicenow.com/) — Rapidly build, test, and deploy applications that make work better for your organization. Free Instance & access early previews.
   * [simperium.com](https://simperium.com/) — Move data everywhere instantly and automatically, multi-platform, unlimited sending and storage of structured data, max. 2,500 users/month
   * [stackstorm.com](https://stackstorm.com/) — Event-driven automation for apps, services and workflows, free without flow, access control, LDAP,...
-  * [streamdata.io](https://streamdata.io/) — Turns any REST API into an event-driven streaming API. Free plan up to 1 million messages and 10 concurrent connections
+  * [streamdata.io](https://streamdata.io/) — Turns any REST API into an event-driven streaming API. Free plan up to 1 million messages and 10 concurrent connections.
+  * [Supabase](https://supabase.io) — The Open Source Firebase Alternative to build backends. Free Plan offers the following :
```
Comment (ripienaar) [inline README.md:800]: can be one line really, I prefer one line else the list can never be sorted without huge effort.

File: README.md [hunk 3]
```diff
@@ -902,9 +915,11 @@ Table of Contents
 
 ## Issue Tracking and Project Management
 
+   * [acunote.com](https://www.acunote.com/) — Free project management and SCRUM software for up to 5 team members
    * [acunote.com](https://www.acunote.com/) — Free project management and SCRUM software for up to 5 team members
```
Comment (cmmata) [inline README.md:919]: This line is duplicated

File: README.md (hunk 1 above)
Comment (thispsj) [reply README.md:779]: The Cloud Firestore,Cloud Functions & Big Query are a part of GCP.
But the other services are exclusively a part of Firebase (although they're hosted on GCP)

File: README.md (hunk 3 above)
Comment (thispsj) [reply README.md:919]: Yup I noticed. I think it's my editor's mistake.

File: README.md (hunk 2 above)
Comment (thispsj) [reply README.md:800]: Ok I'll make both Firebase & Supabase short.

**Comment by ripienaar** [comment]
thanks!

**Pull request merged**